interface_JS/interface_tableau.js

```diff
@@ -165,7 +165,16 @@ function getOutputGenOptions() {
 
     //plug correct value into category options
     genOptions.rootCategory = spotForm['genOptions-rootCategory'].value;
-    genOptions.recursiveCategory = spotForm['genOptions-recursiveCategory'].value;
+    genOptions.recursiveCategory = "";
+    genOpsRC = spotForm['genOptions-recursiveCategory'];
+    for (var i = 0; i<genOpsRC.length; i++){
+        if(genOpsRC[i].value && genOpsRC[i].checked){
+            if(genOptions.recursiveCategory.length){
+                genOptions.recursiveCategory = genOptions.recursiveCategory.concat("-")
```

Comment:
I take it you decided not to refactor options.recursiveCategory into an array.

Reply:
It is true.  The variable options.recursiveCategory remains a string, but recCats is always an array.  I can't remember the exact issue that caused me to move away from changing recursiveCategory, but it seemed simpler to just make sure recCats was an array.

Reply:
Ok. I'll mark that for a refactor later.